packages/apify/src/actor.ts

```diff
@@ -399,7 +396,7 @@ export class Actor<Data extends Dictionary = Dictionary> {
             customAfterSleepMillis = this.config.get('metamorphAfterSleepMillis'),
             ...metamorphOpts
         } = options;
-        const runId = this.config.get('actorRunId')!;
+        const runId = process.env[ENV_VARS.ACTOR_RUN_ID]!;
```

Comment:
Why are we coming back to reading process envs? I thought the configuration would read the envs and then everything else would read it from configuration.

Reply:
Removed those changes for now, I guess we should have that second config class in actor sdk if we want to support those too.